apps/prairielearn/src/tests/sharedQuestionPreview.test.ts

```diff
@@ -82,11 +81,32 @@ describe('Shared Question Preview', function () {
       isStudentPage: false,
     };
 
-    testQuestionPreviews(previewPageInfo, addNumbers, addVectors);
-
-    testFileDownloads(previewPageInfo, downloadFile, false);
-
-    testElementClientFiles(previewPageInfo, customElement);
+    describe('When questions are share_source_publicly but not shared_publicly', async function () {
+      before(
+        'Make sure question have share_source_publicly set but not shared_pubilcly',
+        async function () {
```

Comment:
Nit: typically we'd reach for arrow functions here unless access to `this` is specifically needed. That said, I imagine you're just matching against other tests in this same file.

Reply:
I just went ahead and updated the whole file, it was pretty simple.